makefile

```diff
@@ -20,6 +20,7 @@ TARGET_PATH    := ../$(TARGETS_FOLDER)/$(TARGET)
 APP_PATH       := ../$(APPS_FOLDER)/$(APP)
 LIB_PATH       := ../lib
 MBED_PATH      := ../mbed-os
+HW_BRIDGE_PATH := ../uwrt_mars_rover_hw_bridge/config
```

Comment:
don't think we need it to be in a config subfolder. should change this upstream too